packages/specs/schema/src/components/default/discriminatorMappingMapper.ts

```
@@ -0,0 +1,37 @@
+      newMapping[key] = isString(value) ? value : toRef(value, null, options).$ref;
+    }
+
+    obj.discriminator.mapping = newMapping;
+  }
+
+  return obj;
+}
+
+function defaultDiscriminatorMappingMapper(obj: SchemaWithDiscriminator) {
+  if (obj.discriminator?.mapping) {
+    delete obj.discriminator.mapping;
```

Biome

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)